lib/vagrant-sshfs/cap/guest/linux/sshfs_forward_mount.rb

```diff
@@ -173,8 +220,11 @@ def self.sshfs_slave_mount(machine, opts, hostpath, expanded_guest_path)
           #
           # Wire up things appropriately and start up the processes
           if Vagrant::Util::Platform.windows?
-            # Need to handle Windows differently. Kernel.spawn fails to work, if the shell creating the process is closed.
-            # See https://github.com/dustymabe/vagrant-sshfs/issues/31
+            # For windows we need to set it so not all file handles are inherited
+            # by default. See https://github.com/dustymabe/vagrant-sshfs/issues/41
+            self.windows_uninherit_handles
```

Comment:
Since we want it to be an independent process I'd rather only inherit the things that we want to rather than everything Vagrant has open at the time we spawn off the processes. Just seems cleaner.